tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/IReceiveMessageResponse.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="IReceiveMessageResponse.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections;
+using System.Collections.Generic;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.SQS
+{
+    internal interface IReceiveMessageResponse : IDuckType
+    {
+        IList Messages { get; } // <IMessage>
+    }
+
+    internal interface IMessage : IContainsMessageAttributes
```

Comment:
I think that rule is for classes, but multiple interfaces in one file are ok.
I thought this helped with readability since it's defined only for the usage that's just above, but I can split it in 2 files if you think it's better (or make IMessage a sub-type of `IReceiveMessageResponse` ?)

Reply:
> I think that rule is for classes, but multiple interfaces in one file are ok.

Oh ok, TIL 😅  FWIW, I think it's fine having these together, it's just not _common_ in .NET and can sometimes make it harder to find type definitions (depending on your IDE).